README.md

```diff
@@ -33,6 +33,9 @@ This will start up the build-process with hot-reload.
 To add the extension to the browser, navigate to `chrome://extensions/` and click `Load unpacked`.
 Navigate and add the `dist/` folder of this project.
 
+To publish the extension to the Chrome Web Store, upload `releases/solid-identity-manager.zip` to the store.
```

Comment:
- Can we have dedicated section for preparing the extension for publishing it to the store? 
- Can you include a numerated list with steps?

Reply:
And add a link to the Web Store docs on how to setup an account in case they haven't got one yet?

Reply:
Yes, add a link to the docs for creating an account.